css/excercise_one/excercise_one.css

```diff
@@ -0,0 +1,65 @@
+* {
```

Comment:
good work here! don't know if background-color is needed or text-decoration.

Reply:
Apparently, by default, <a> tags are underlined. That's why
Background color is not necessary

Reply:
you can use `font-style: normal` that makes the style of the font without anything, just in case.